examples/disaster-recovery-example/README.md

```diff
@@ -0,0 +1,80 @@
+# Disaster Recovery Example
+This provides an example of the code and knowledge required to recover documents encrypted using SaaS Shield in the case of catastrophic IronCore Labs failure. This should never have to happen, even in the event of stopping usage of IronCore you will still be able to migrate documents out of the system more easily than this. This is pretty strictly an "IronCore and all their services were hit by a meteor storm" or "I'm an archeologist trying to recover lost encrypted data" situation.
```

Comment:
```suggestion
This provides an example of the code and knowledge required to recover documents encrypted using SaaS Shield in case IronCore's Configuration Broker is unavailable and data must be accessed before it is likely to be accessible again. This should never have to happen. Even in the event of stopping usage of IronCore, you will still be able to migrate documents out of the system without resorting to a recovery effort. This is to satisfy those who are worried about "IronCore and all their services were hit by a meteor storm" or "I'm an archeologist trying to recover lost encrypted data" situations.
```

Reply:
I'm fine with the wording, but it is more extreme than that, in the sense that this assumes they can't even access a TSP or a TSC for that matter. It'd be slightly easier using some helper functions that are in the TSC, and possible you could unwrap using leasing/caching if it's just that the config broker is unavailable. This code/example is fully written with the idea that we don't exist in any way, shape, or form anymore.

Reply:
Good point. I still want to soften that a little bit even in that intro. What I said is still accurate, I think, provided you don't have any running TSCs.

Reply:
It's not accurate, we could make it "IronCore's Configuration Broker is unavailable and data must be accessed before it is likely to be accessible again, all of your TSPs are down, and you've lost access to the TSC package."

I'm not sure I follow wanting to soften it though, to me it makes sense to stress it has to be a total and complete failure of some sort for this to be necessary.

Reply:
I actually like your proposed additions since it highlights how many things have to go wrong for this scenario to be plausible.